### PR TITLE
[LISTING] - Sepolia: FUND & FUNDx

### PR DESCRIPTION
**Pull request by tokdaniel**

### CODE REVIEW

File: superfluid.extended.tokenlist.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": {
     "major": 2,
     "minor": 0,
```
Comment (elvijsTDL) [inline superfluid.extended.tokenlist.json:5]: Shouldn't the major version be 3 here too?

File: superfluid.tier-a.tokenlist.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": {
     "major": 2,
     "minor": 0,
-    "patch": 0
+    "patch": 1
```
Comment (elvijsTDL) [inline superfluid.tier-a.tokenlist.json:6]: Same thing about major version here?

File: superfluid.tier-b.tokenlist.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "version": {
     "major": 2,
     "minor": 0,
-    "patch": 0
+    "patch": 1
   },
```
Comment (elvijsTDL) [inline superfluid.tier-b.tokenlist.json:7]: And here

File: superfluid.tier-c.tokenlist.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
   "version": {
     "major": 2,
     "minor": 0,
-    "patch": 0
+    "patch": 1
```
Comment (elvijsTDL) [inline superfluid.tier-c.tokenlist.json:6]: And here

File: superfluid.tokenlist.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
   "version": {
     "major": 2,
     "minor": 0,
-    "patch": 0
+    "patch": 1
```
Comment (elvijsTDL) [inline superfluid.tokenlist.json:6]: And here

File: superfluid.extended.tokenlist.json (hunk 1 above)
Comment (tokdaniel) [reply superfluid.extended.tokenlist.json:5]: nope, that's the package, it can have changes without changing the tokenlist.

**Review by kasparkallas — APPROVED** [review]

**Pull request merged**